dashboard/plugins/table_descriptions/dataproviders.py

```diff
@@ -23,20 +23,20 @@ def __exit__(self, *args):
 
 class GlueDescriptionService:
 
-    def __init__(self, logger, config, tables):
+    def __init__(self, logger, config, tables, region_name, aws_access_key_id, aws_secret_access_key):
         self.client = boto3.client('glue',
-                                   region_name=config['GLUE_REGION_NAME'],
-                                   aws_access_key_id=config['AWS_ACCESS_KEY_ID'],
-                                   aws_secret_access_key=config['AWS_SECRET_ACCESS_KEY'])
+                                   region_name=region_name,
+                                   aws_access_key_id=aws_access_key_id,
+                                   aws_secret_access_key=aws_secret_access_key)
         self.logger = logger
         self.db_tables = self.get_db_tables(tables)
 
 
     @staticmethod
     def get_db_tables(tables):
         result = defaultdict(lambda: list())
```

Comment:
Not your change, but Tomek once pointed out you can just provide class name instead of lambda with factory method.

Reply:
Right, thanks!